endpoints.json

```diff
@@ -54,20 +54,35 @@
 			}
 		}
 	},
-  "GET /api/articles/:article_id/comments": {
+	"GET /api/articles/:article_id/comments": {
 		"description": "serves an array of the comments specified by article id",
 		"queries": ["article_id"],
 		"exampleResponse": {
-			"comments": [{
+			"comments": [
+				{
+					"article_id": 1,
+					"title": "Living in the shadow of a great man",
+					"topic": "mitch",
+					"author": "butter_bridge",
+					"body": "I find this existence challenging",
+					"created_at": "2020-07-09T20:11:00.000Z",
+					"votes": 100,
+					"article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+				}
+			]
+		}
+	},
+	"POST /api/articles/:article_id/comments": {
+		"description": "posts an object of comment based on article id returns the posted object",
+		"queries": ["article_id", "req.body"],
+		"exampleResponse": {
+			"newComment": {
+				"comment_id": 1,
 				"article_id": 1,
-				"title": "Living in the shadow of a great man",
-				"topic": "mitch",
 				"author": "butter_bridge",
-				"body": "I find this existence challenging",
-				"created_at": "2020-07-09T20:11:00.000Z",
-				"votes": 100,
-				"article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
-			}]
+				"votes": 0,
+				"created_at": "2020-07-09T20:11:00.000Z"
+			}
 		}
 	}
```

Comment:
Great to see this added to the endpoints.json file as well, but dont forget to include the example body we expect to receive from the user as well.